modules/gitops-inbuilt-permissions-for-cluster-config.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assembly:
+//
+// * configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
+
+:_content-type: REFERENCE
+[id="gitops-inbuilt-permissions-for-cluster-config_{context}"]
+= In-built permissions for cluster configuration
+
+By default, the Argo CD instance has permissions to manage specific cluster-scoped resources such as platform Operators, optional OLM Operators and user management. 
+
+[NOTE]
+====
+Argo CD does not have cluster-admin permissions.
+====
+
+Permissions for the Argo CD instance:
+|===
```

Comment:
openshift-docs prefers table formatting like this: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#alternative-footnote-styling-in-tables